runtime/src/accounts_db.rs

```diff
@@ -10469,9 +10469,9 @@ pub mod tests {
         account_data_size: Option<u64>,
     ) -> Arc<AccountStorageEntry> {
         let (_temp_dirs, paths) = get_temp_accounts_paths(1).unwrap();
-        let size: usize = 123;
+        let size: usize = aligned_stored_size(account_data_size.unwrap_or(123) as usize);
         let mut data = AccountStorageEntry::new(&paths[0], slot, id, size as u64);
-        let av = AppendVec::new(&tf.path, true, 1024 * 1024);
+        let av = AppendVec::new(&tf.path, true, (1024 * 1024).max(size));
```

Comment:
what's the reason for imposing a minimum appendvec size here?

Reply:
cause that's how it's been. Just making sure we get an append vec of big enough size so tests pass. Could probably be made much smaller. in my case, I needed bigger than this. I can look into changing this in a follow on pr to fix the status quo.